### PR TITLE
(chore)refactor: Converting the component names and spec attributes in desired formate

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: Udit Gaurav <udit.gaurav@mayadata.io>

**What this PR does / why we need it**:

This is Pr contains the script for some correcting the formate of texts in the docs. This include:
1. It will Identify wherever we are using resource names like ChaosEngine, ChaosExperiment, ChaosResult, ChaosSchedule etc., convert it in PascalCase.

2. It will Identify wherever we are using spec attributes and it converts them to camelCase.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

How to run it?

```
python3 hack/formate.py
```

**Checklist:**
-   [ ] Fixes #<issue number>
-   [ ] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

**Comment by divya-mohan0209** [comment]
@uditgaurav : While I feel like this is a good step ahead in automating the efforts, we need to ensure that everything doesn't get converted to a PascalCase/camelCase in the documentation. I did see some examples that I'd like to discuss in the Docs meeting today, post which we shall approve/suggest improvements.

File: docs/chaoshub.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ sidebar_label:ChaosHub
 
 Chaos Hub is maintained at https://hub.litmuschaos.io
 
-To contribute new chaos charts visit: https://github.com/litmuschaos/chaos-charts
+To contribute new ChaosCharts visit: https://github.com/litmuschaos/chaos-charts
 
 **Introduction**
 
-Litmus chaos hub is a place where the chaos engineering community members publish their chaos experiments. A set of related chaos experiments are bundled into a `Chaos Chart`. Chaos Charts are classified into the following categories.
+Litmus chaos hub is a place where the ChaosEngineering community members publish their chaos experiments. A set of related chaos experiments are bundled into a `Chaos Chart`. Chaos Charts are classified into the following categories.
```
Comment (uditgaurav) [inline docs/chaoshub.md]: ChaosEngineering should not be converted!

**Comment by uditgaurav** [comment]
@divya-mohan0209 @ksatchit I have removed the automation script for now. We can take this as a refactor PR. I will add the automation script with some more enhancements.

**Review by divya-mohan0209 — COMMENTED** [review]
@uditgaurav : Does this mean that the changes are currently being made only to the documentation & you'll be updating the refactoring script per our conversation later on?

**Comment by uditgaurav** [comment]
>Does this mean that the changes are currently being made only to the documentation & you'll be updating the refactoring script per our conversation later on?

Yes @divya-mohan0209

**Review by divya-mohan0209 — APPROVED** [review]
LGTM. Approved. @ksatchit @k8s-dev : Please review for any further changes.

**Review by ksatchit — APPROVED** [review]

**Pull request merged**